components/ArtPiecePreview/index.js

```diff
@@ -0,0 +1,13 @@
+export default function ArtPiecePreview({ imageSource, title, artist }) {
+  return (
+    <div>
```

Comment:
" Create the component ArtPieces to render a list" - this indicates we need a `<ul><li></li><li></li></ul` - structure...better let this div be a li